controllers/managedocs_controller.go

```diff
@@ -911,12 +911,18 @@ func (r *ManagedOCSReconciler) reconcileAlertRelabelConfigSecret() error {
 		}
 
 		alertRelabelConfig := []struct {
-			TargetLabel string `yaml:"target_label,omitempty"`
-			Replacement string `yaml:"replacement,omitempty"`
-		}{{
-			TargetLabel: "namespace",
-			Replacement: r.namespace,
-		}}
+			SourceLabels []string `yaml:"source_labels"`
+			TargetLabel  string   `yaml:"target_label,omitempty"`
+			Replacement  string   `yaml:"replacement,omitempty"`
+		}{
+			{
+				SourceLabels: []string{"namespace"},
+				TargetLabel:  "alertnamespace",
+			},
+			{
+				TargetLabel: "namespace",
+				Replacement: r.namespace,
+			}}
```

Comment:
Why are we moving the source namespace to alertnamespace? Can we just keep the sourcenamespace at `namespace` and have `r.namespace` be in the `alertnamepsace`. To me, it makes much more sense do as little changes to incoming metric data. If we need additional labels we just add them.

Reply:
Our alertmanager uses the `namespace: openshift-storage`matcher to catch all the alerts, if we want to change `namespace: openshift-storage`, then we will also have to change `alertnamespace: openshift-storage` in alertmanager too

```
matchers:
    - namespace="openshift-storage"
```

Reply:
Changing the selector seems to me like a more reasonable change

Reply:
But in both cases, i don't think `alertnamespace` is a name that describes the data we are adding. If the only concern we have is that matcher why can't we just add our common= label `app=managed-ocs`?

Reply:
Right now we don't have this label in the alert, but anyways we can add it if you suggest and we will keep `namespace` as it is

Reply:
I think we cannot change the match label for alertmanager, we need to make changes in relabelConfig only.